src/ol/source/imagesource.js

```diff
@@ -47,11 +46,6 @@ ol.source.Image = function(options) {
    */
   this.resolutions_ = goog.isDef(options.resolutions) ?
       options.resolutions : null;
-  goog.asserts.assert(goog.isNull(this.resolutions_) ||
-      goog.array.isSorted(this.resolutions_,
-          function(a, b) {
-            return b - a;
-          }, true));
```

Comment:
The `resolutions` option has to do with the discrete resolutions at which a source is available.  By removing this assertion, you are changing the behavior of other sources that extend `ol.source.Image`.